README.md

```diff
@@ -21,6 +21,51 @@ _**Note**: below listed features are still under design and review phase and hen
 * New range functions
 * Freeze rows/columns.
 
+## Upcoming Excel 1.8 Event Release Features
```

Comment:
FYI, our style is to use sentence case for headings. This would apply globally to all the headings in this readme file - i.e. "Upcoming Excel 1.8 event release features". Also do we need to say "release", or are they just "event features"?